pixart.js

```diff
@@ -1 +1,24 @@
+// // Like the A-Team, but with variables
+var setColorButton = $('#set-color');
+var inputField= $('#color-field');
+var brushElement = $('.brush');
+var brushColor = "#1B4370";
 
+setColorButton.click(setBrush);
+
+function setBrush() {
+  event.preventDefault();
+  brushElement.css("background-color", inputField.val()) ;
+}
+
+for (i=0; i < 8000; i++){
+  $('body').append("<div class='square'> </div>");
+}
+$(".square").on("mouseover", setSquareColor);
+
+function setSquareColor () {
+  if (brushElement.css("background-color")) {
+    brushColor=brushElement.css("background-color");
```

Comment:
Also, this line is retrieving the color (i.e. some data) from the DOM. A better approach might be to use a shared JS variable (`currentColor`) that you set in `setBrush` and read from here in `setSquareColor`.